shards/modules/gfx/shards_types.hpp

```diff
@@ -340,7 +340,7 @@ struct Container {
 
   static inline ParameterInfo OutputScaleParameterInfo{"OutputScale",
                                                        SHCCSTR("The scale that the output should be rendered as"),
-                                                       {CoreInfo::NoneType, CoreInfo::Float2Type, CoreInfo::Float2VarType}};
+                                                       {CoreInfo::NoneType, CoreInfo::AnyType}};
```

Comment:
seems weird?

Reply:
Didn't write tests for it yet but the new :OutputScale parameter is either:
`{main: none}` => Scale to main output
`none` =>      Same as above
`{main: @f2(0.25)}` => Scale to 25% of main output
`{main: 0.25}` =>  Same as above
`@f2(0.25)` => Same as above
`{input: "color" scale: @f2(0.25)}` => Scale to 25% of the color input

Had too many combinations and optionals to provide the type